### DISCUSSION TITLE
Have a real grid (table) container that you add a control node in any location.

### DISCUSSION THREAD

**Discussion by cloa513** [godotengine/godot-proposals#4403] [GUI]
There are some tables for data control but if you want a table to arrange control nodes- as you see in lots of places- then you are forced to  use grid container which is so clumsy as it limited to adding nodes in a compact pattern and really difficult if you want to change its number of columns. The table would have set the number of rows and column and their widths and heights.  Be able to merging the cells of the table would nice too.

**Comment by Xrayez** [comment]
See #97.

**Comment by cloa513** [comment]
That is a data one- already one created like that. Lots of button in grids in lots of websites and games.
